eth/tracers/api_test.go

```diff
@@ -198,13 +198,15 @@ func TestTraceCall(t *testing.T) {
 	}
 	genBlocks := 10
 	signer := types.HomesteadSigner{}
-	api := NewAPI(newTestBackend(t, genBlocks, genesis, func(i int, b *core.BlockGen) {
+	backend := newTestBackend(t, genBlocks, genesis, func(i int, b *core.BlockGen) {
```

Comment:
It feels weird to call chain stop here. Maybe we should define a teardown function for the backend and call defer backend.teardown after initializing backend?